device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/InternalClient.java

```diff
@@ -862,206 +848,151 @@ void setDeviceIO(DeviceIO deviceIO)
         }
     }
 
-    void setOption_SetHttpsConnectTimeout(Object value)
+    private void setOption_SetHttpsConnectTimeout(Object value)
     {
-        if (value != null)
+        if (this.config.getProtocol() != HTTPS)
         {
-            if (this.config.getProtocol() != HTTPS)
-            {
-                throw new UnsupportedOperationException("Cannot set the https connect timeout when using protocol " + this.config.getProtocol());
-            }
+            throw new UnsupportedOperationException("Cannot set the https connect timeout when using protocol " + this.config.getProtocol());
+        }
 
-            if (value instanceof Integer)
-            {
-                log.info("Setting HTTPS connect timeout to {} milliseconds", value);
-                this.config.setHttpsConnectTimeout((int) value);
-            }
-            else
-            {
-                throw new IllegalArgumentException("value is not int = " + value);
-            }
+        if (value instanceof Integer)
+        {
+            log.info("Setting HTTPS connect timeout to {} milliseconds", value);
+            this.config.setHttpsConnectTimeout((int) value);
+        }
+        else
+        {
+            throw new IllegalArgumentException("value is not int = " + value);
         }
     }
 
-    void setOption_SetHttpsReadTimeout(Object value)
+    private void setOption_SetHttpsReadTimeout(Object value)
     {
-        if (value != null)
```

Comment:
These values were already null checked in the method that calls this method, so I'm removing them